shell/detail/provisioning.cattle.io.cluster.vue

```diff
@@ -321,6 +321,14 @@ export default {
       return false;
     },
 
+    showEksNodeGroupWarning() {
+      if ( this.value.isEKS && this.nodes?.length < 1 ) {
+        return !!this.value.eksNodeGroups.find(g => g.desiredSize === 0);
```

Comment:
It looks like this is returning `true` (showing the banner) if there are 0 nodes in _any_ node group but (assuming I've understood the issue correctly; I didn't read the entire thread in the original issue) it should only show if there are 0 nodes available in _all_ node groups.

Reply:
Good catch, I updated the method to filter for _all_ node groups and check against that total number.